src/components/App.js

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from "react";
+import FolderList from "./FolderList";
+import { getFolders, getUser } from "../api";
+import Header from "./Header";
+import Footer from "./Footer";
+import Profile from "./Profile";
+import SearchBar from "./SearchBar";
+
+function App() {
+  const [items, setItems] = useState([]);
+  const [user, setUser] = useState(null);

```

Comment:
이런 데이터를 다룰때 웬만하면 디폴트값을 주는게 안전합니다 . user 객체의 기본적인 구조를 알고있다면 빈스트링이나 0과 같은 디폴트값을 주세요. 
또는 user 객체를 사용하는 컴포넌트에서 nullable 체크를 한 뒤 디폴트 값을 주도록 해주세요.

```suggestion
  const [user, setUser] = useState({
     email: "",
     profileImageSource: "",
  });
```

```javascript
function userInformation({ user }) {
  if (!user) return false;
  return (
    <div>
      <img className="logo" src={user.profileImageSource || ""} alt={profileImageSource} />
      <span>{user.email || ""}</span>
    </div>
  );
}
```

alt값은 이미지의 주소값이 아닌 이미지를 나타 낼 수 있는 문장이나 단어를 써주세요.
예를들면 `프로필 이미지`  처럼요